quarkus-jnosql-mongodb/pom.xml

```diff
@@ -59,6 +60,17 @@
       <artifactId>rest-assured</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.github.javafaker</groupId>
```

Comment:
why a old version of Java faker?

https://mvnrepository.com/artifact/com.github.javafaker/javafaker/1.0.2

Reply:
Oh no... I'll fix it! Thanks, @otaviojava!